cypress/integration/PrettyUrlRedirect.feature

```diff
@@ -0,0 +1,7 @@
+Feature: As the website, I should redirect CDR IDs in URLs to its counterpart pretty URL for better findability
+
+
+    Scenario: Redirect from CDR ID
+        Given the user navigates to "/def/750633"
+        Then the system redirects user to "/def/acenocoumarol"
+        And the system appends "?redirect=true" to the URL
```

Comment:
there's that symbol again!

Reply:
Weird - I ran **npm run format** and this was not in the list of formatted files

Reply:
Removed